group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupConfigTest.java

```diff
@@ -57,8 +57,6 @@ public void testFromPropsInvalid() {
                 assertPropertyInvalid(name, "not_a_number", "-0.1", "1.2");
             } else if (GroupConfig.SHARE_RECORD_LOCK_DURATION_MS_CONFIG.equals(name)) {
                 assertPropertyInvalid(name, "not_a_number", "-0.1", "1.2");
-            } else {
-                assertPropertyInvalid(name, "not_a_number", "-1");
             }
```

Comment:
Why this change?

Reply:
Thanks for the review. Due to the repeated iterations of the code, there was a time when this was actually failing the test because but now this line should be there. I will add this in the next commit